### ISSUE TITLE
LocalStorage pollution after SSO

### ISSUE THREAD

**Issue by akirk** [Automattic/chatrix#57]
`hydrogen_setting_v1_sso_ongoing_login_homeserver` remaings lying in localStorage after login process is complete

**Comment by psrpinto** [comment]
This would have to be fixed in hydrogen, I could open a PR. Do you think it's worth doing @akirk?
I would be ok with leaving it as is since it has no functional impact I can think of :)

**Comment by ashfame** [comment]
+1 We can file an issue and move on.

**Comment by akirk** [comment]
I agree with filing an issue for now.

**Comment by psrpinto** [comment]
I feel a bit guilty opening an issue on hydrogen for this as I feel it's a very small detail that has no functional impact, and hydrogen has a large backlog of issues. We would also be spending some "maintainer credits" which might make more sense to dedicate to more pressing issues, for example our open PRs.

So I looked into what it would take to fix it, since it felt like an easy fix. However, upon investigation, doing it "properly" would involve refactoring some code, mostly due to having to handle non-happy paths. So I don't think it would indeed make sense for us to invest effort into this.

So before opening an issue in hydrogen, I figured I would ask here again whether you think it's worth pursuing this, or whether we could drop it. If you think we should indeed open an issue, I'll be happy to do so.
